### PR TITLE
[HackerRank] Pairs

### PR DESCRIPTION
**Pull request by swarn4399**
Python solution for the "Pairs" problem in the Algorithm section of Hackerrank.

### CODE REVIEW

**Comment by aneels3** [comment]
@swarn4399 change the PR name as mentioned in the contribution guideline.

**Comment by swarn4399** [comment]
@aneels3 Sir, "Pairs" is the name of the problem on Hackerrank.

File: HackerRank Problem/Python/Pairs.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    fptr = open(os.environ['OUTPUT_PATH'], 'w')
+
+    nk = raw_input().split()
+
+    n = int(nk[0])
+
+    k = int(nk[1])
+
+    arr = map(int, raw_input().rstrip().split())
+
```
Comment (aneels3) [inline HackerRank Problem/Python/Pairs.py]: please remove the unnecessary extra lines in your code

Comment (swarn4399) [reply HackerRank Problem/Python/Pairs.py]: Should i remove the problem statement too?

Comment (aneels3) [reply HackerRank Problem/Python/Pairs.py]: No, Its fine

Comment (aneels3) [reply HackerRank Problem/Python/Pairs.py]: delete the extra lines for example line number 58, 60, 62 and so on...

**Comment by swarn4399** [comment]
@aneels3 I updated it. Pls let me know your remarks.

File: HackerRank Problem/Python/Pairs.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+if __name__ == '__main__':
+    fptr = open(os.environ['OUTPUT_PATH'], 'w')
+
+    nk = raw_input().split()
+
+    n = int(nk[0])
+
+    k = int(nk[1])
+
+    arr = map(int, raw_input().rstrip().split())
```
Comment (aneels3) [inline HackerRank Problem/Python/Pairs.py]: you are not getting my point. After taking the inputs like nk, n, k, arr there are extra lines. remove those only

File: HackerRank Problem/Python/Pairs.py [hunk 3]
```diff
@@ -17,13 +17,13 @@ def pairs(k, arr):
 if __name__ == '__main__':
     fptr = open(os.environ['OUTPUT_PATH'], 'w')
 
-    nk = raw_input().split()
+    '''nk = raw_input().split()
 
     n = int(nk[0])
 
     k = int(nk[1])
```
Comment (aneels3) [inline HackerRank Problem/Python/Pairs.py]: n = int(nk[0])

k = int(nk[1])

these lines should be like this
n = int(nk[0])
k = int(nk[1])

without any extra lines between them.

**Comment by aneels3** [comment]
@swarn4399 great work!

**Pull request merged**